### PR TITLE
Websocket logs endpoint for polling envoy fleet container logs

### PR DESCRIPTION
**Pull request by kylehodgetts**
Changes
- add websocket handler and client to api
- add gorilla websocket as dependency for serving the websocket
- update websocket container to build and release pipelines

when starting websocket process you can specify the following options
```
...
flag.IntVar(&port, "port", 8080, "port to listen on")
flag.IntVar(&writeWaitSecs, "write-wait", defaultWriteWaitSecs, "time allowed to write a message to the peer")
flag.IntVar(&pongWaitSecs, "pong-wait", defaultPongWaitSecs, "time allowed to read the next pong message from the peer")
flag.Int64Var(&maxMessageSize, "max-message-size", defaultMaxMessageSize, "maximum message size allowed from peer")
...
```

when a request for envoy fleet logs comes in, a container log stream for the envoy container in the requested envoy fleet deployment is fetched from the api and the new logs that are sent over the stream are then sent over the websocket.

### CODE REVIEW

**Review by jasmingacic — APPROVED** [review]
I know it is a draft but It is looking good to be packaged. 

Wrap up the docker file and add it to https://github.com/kubeshop/kuskgateway-api-server/blob/main/kgw-api.yaml and we are good to go

**Review by jasmingacic — APPROVED** [review]
This is looking good. 

Ready to be shipped

File: api/openapi.yaml [hunk 1]
```diff
@@ -425,6 +425,45 @@ paths:
...
+          schema:
+            type: string
+            default: "kusk-gateway-envoy-fleet"
+      responses:
+        200:
+          description: "Envoy fleet logs"
+          content:
+            application/json:
+              schema:
+                type: object
```
Comment (mortada-codes) [inline api/openapi.yaml:463]: can we change the type to an array of string? because it generates a wrong type

Comment (kylehodgetts) [reply api/openapi.yaml:463]: What's `it`? Are you using a library on the front end to generate stuff?

**Review by jasmingacic — APPROVED** [review]

**Pull request merged**